.github/workflows/docs-site.yml

```diff
@@ -0,0 +1,40 @@
+name: Build & Deploy Docs Site (on-merge)
+
+on:
+  push:
+    paths:
+    - 'docs/**'
+    branches:
+      - master
+  pull_request:
+    paths:
+    - 'docs/**'
+    types: [closed]
+
+jobs:
+  docs:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@master
+        with:
+          python-version: 3.6
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
```

Comment:
The location of mkdocs requirements.txt conflicts with projects  requirements.txt.
Is it a good idea to move it to docs folder ? What do you think is the best approach ?

Reply:
Added a separate `mkdocs-requirements.txt` for mkdocs related dependencies